src/styles/blocks/container.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $full-width;
+  gap: $space;
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
```

Comment:
Will be better to specify this property via `*` selector as it will be applied as the general rule for all elements in your project